__tests__/components/index.test.js

```diff
@@ -0,0 +1,3 @@
+it('Should pass', () => {
+  expect(5 * 5).toEqual(25);
+});
```

Comment:
Why did you name test folder like that, with underscores

Reply:
Underscores and double underscores are commonly used to denote things that are outside of the regular code.